libnmstate/varlink/__init__.py

```diff
@@ -0,0 +1,313 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+import os
+import libnmstate
+import argparse
+import logging
+import pkg_resources
+import libnmstate.error as libnmError
+
+
+try:
+    import varlink
+
+    if pkg_resources.get_distribution("varlink").version < "30.2.0":
```

Comment:
Any reason we need this version of `varlink`?

And this version is not shipped by RHEL/CentOS 8. If we do that, we lose the ability to ship this service in RHEL 8 which is not acceptable for nmstate users.

Reply:
This is the deal breaker. Please support existing RHEL/CentOS 8 python3-varlink 29.0.0.

Reply:
And also consider the SLES is also shipping dated `python3-varlink`.